src/general/NewGameSettings.cs

```diff
@@ -384,10 +389,13 @@ public void OpenFromDescendScreen(GameProperties currentGame)
         playerDeathPopulationPenalty.Value = difficulty.PlayerDeathPopulationPenalty;
         glucoseDecayRate.Value = difficulty.GlucoseDecay * 100;
         osmoregulationMultiplier.Value = difficulty.OsmoregulationMultiplier;
+        fogOfWarModeDropdown.Selected = (int)difficulty.FogOfWarMode;
```

Comment:
I assume this fixes a mistake that was in a previous PR.

Reply:
Indeed